### ISSUE TITLE
How to build jythonc from source

### ISSUE THREAD

**Issue by hailijuan** [jython/jython#46]
I want to use jythonc compile java class file independently. Is there any documentation on how to build jythonc from source? Thanks a lot!

**Comment by jeff5** [comment]
Not really. The source is at src/org/python/util/JythoncAntTask.java, but is only used within our ant build to process the stdlib. It gets built along with every thing else in the Java compilation stage of the ant build.

A Jython `$py.class` depends heavily on the Jython runtime, as you would expect, so it is not much use separately. Depending on your purpose, you may be interested in [Clamp](https://github.com/jythontools/clamp/).
